design-reference/architecture/aad/easyauth.md

```diff
@@ -0,0 +1,35 @@
+# Purpose/Description
+
+Cobalts customers expect the ability to configure Authentication for their applications. We've chosen to provide this ability by extending both the Service Principal module and the App Service module. This will enable a customer to configure "Easy Auth" on their App Service using an application registration in Azure Active Directory.
+Azure App Service provides built-in authentication and authorization support, so you can sign in users and access data by writing minimal or no code in your web app, RESTful API, and mobile back end, and also Azure Functions. This article describes how App Service helps simplify authentication and authorization for your app.
```

Comment:
You may want to provide some guidance for how REST clients will set the auth token header?